Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -52,8 +52,18 @@ WebFoundTextRangeController::WebFoundTextRangeController(WebPage& webPage)
 {
 }
 
+void WebFoundTextRangeController::presentFindNavigatorWithoutReplace()
+{
+    m_webPage->presentFindNavigatorWithoutReplace();
+}
+
 void WebFoundTextRangeController::findTextRangesForStringMatches(const String& string, OptionSet<FindOptions> options, uint32_t maxMatchCount, CompletionHandler<void(Vector<WebFoundTextRange>&&)>&& completionHandler)
 {
+#if ENABLE(IMAGE_ANALYSIS)
+    m_webPage->corePage()->analyzeImagesForFindInPage([this] {
```

Comment:
This should capture a `WeakPtr`, there's no guarantee the `WebFoundTextRangeController` will still be alive.

Will also need to make the class inherit from `public CanMakeWeakPtr< WebFoundTextRangeController>`.